src/wslsys.sh

```diff
@@ -116,6 +116,10 @@ function get_wsl_packages() {
 	echo "$packages"
 }
 
+function get_wsl_ip() {
+	echo "$(hostname -I)"
```

Comment:
busybox `hostname` command does not have -I parameters. What are the additional required packages for Alpine here?

Reply:
``net-tools` neither so this code does not work for alpine.

Reply:
`ip -4 -o addr show eth0 | awk '{print $4}' | cut -d "/" -f 1` seems to work on AlpineWSL. Will it work?

Reply:
It also works on Debian, Ubuntu, Gentoo, and Kali.

Reply:
It requires `iproute`/`iproute2` packages for some distributions (WLE requires `iproute` and Arch requires `iproute2` in my testing). You should add them in the build scripts, thank you.

Reply:
I have tested the command on all the distributions listed in the `configure.sh` and have updated it accordingly.